spec/features/admin/admins/admin_sort_admins_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'Admin can sort an index of users' do
+
+  let!(:alpha_admin) { FactoryGirl.create(:user, :admin, email: "alpha@example.com") }
+  let!(:beta_admin)  { FactoryGirl.create(:user, :admin, email: "beta@example.com") }
+
+  before do
+    sign_in_as(alpha_admin)
+    click_header_option("Dashboard")
+    click_sidemenu_option("Admins")
+  end
+
+  scenario "Using tabs to show different types of users" do
+    # Order by Ascending first
+    click_link("Email")
+
+    expect_users_to_be_ordered_as(alpha_admin, beta_admin)
+
+    # Order by Descending next
+    click_link("Email")
+    expect_users_to_be_ordered_as(beta_admin, alpha_admin)
+  end
+
+private
+
+  def expect_users_to_be_ordered_as(first, second)
+    within(first_row) { expect(page).to have_content(first.email) }
+    within(second_row) { expect(page).to have_content(second.email) }
+  end
+
+  def first_row
+    nth_row(1)
+  end
+
+  def second_row
+    nth_row(2)
+  end
+
+  def nth_row(num)
+    find("table tbody tr:nth-child(#{num})")
+  end
```

Comment:
This is too much cool to live as a local feature test helper, I would move this guy to `https://github.com/thefrontiergroup/rails-template/blob/feature/expand_user_management/spec/support/feature/support.rb`

I guess I pointed this out in another PR, but don't really remember what we ended up deciding...